docs/acp/user/develop-an-app/develop-an-app.md

```diff
@@ -21,7 +21,7 @@ Before you begin, ensure that you have the following prerequisites in place:
 Make sure you have the Spryker Docker SDK, Git, and an empty repository for your app code.
 
 {% info_block infoBox "Info" %}
-[Download the](https://github.com/spryker-projects/mini-framework) completed example from the Mini-Framework.
+[Download the](https://github.com/spryker-projects/mini-framework/tree/examples/acp/create-an-app-final) completed example from the Mini-Framework.
```

Comment:
There must be empty line after 

```
{% info_block infoBox "Info" %}
```

and empty line before: 

```
{% endinfo_block %}
```

otherwise the links in info block don't work.